src/adapter/tstype.ts

```diff
@@ -51,6 +51,17 @@ interface LQI {
     neighbors: LQINeighbor[];
 }
 
+enum RoutingTableStatus {
```

Comment:
This should probably be done in controller (i.e. adapter passes the plain number)? Will likely require a few tweaks in other drivers, but it would cleanup duplicated logic.

Reply:
Let's do it in a follow up PR since it would touch multiple adapters (out of scope for this PR)